hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/source/TestFileIndex.java

```diff
@@ -55,6 +71,35 @@ public class TestFileIndex {
   @TempDir
   File tempFile;
 
+  public static final DataType DATA_TYPE_BIGINT_PK = DataTypes.ROW(
+          DataTypes.FIELD("uuid", DataTypes.BIGINT()),
+          DataTypes.FIELD("name", DataTypes.VARCHAR(10)),
+          DataTypes.FIELD("age", DataTypes.INT()),
```

Comment:
Can we move the data types to `TestConfigurations`

Reply:
Yes

Reply:
Moved